tools/heavy_measurement_lease.py

```
@@ -0,0 +1,502 @@
+            _log("a timing number measured beside another census is not a slow "
+                 "measurement, it is not a measurement.")
+        _log(f"status={lease.status}: this run supports NO claim. Not zero. None.")
+        lease.release()
+        write_receipt(lease, command, None, args.record, args.embed_into)
+        return EX_TEMPFAIL
+
+    try:
+        lease.set_status(STATUS_MEASURING)
+        # pass_fds keeps the lease descriptor open across the fork/exec, so the
+        # measurement itself is a lease holder for as long as it lives.
+        child = subprocess.Popen(command, pass_fds=(lease.fd,))
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)